Travel Scheduler/View Controllers/MoreOptionViewController.h

```diff
@@ -17,6 +17,11 @@ NS_ASSUME_NONNULL_BEGIN
 @property (strong, nonatomic) UICollectionView *collectionView;
 @property (strong, nonatomic) UIButton *scheduleButton;
 @property (strong, nonatomic) NSMutableArray *selectedPlacesArray;
+@property (strong, nonnull) UISearchBar *moreOptionSearchBarAutoComplete;
+@property (strong, nonatomic) NSArray *filteredPlaceToVisit;
```

Comment:
Just wondering, should filteredPlaceToVisit be an NSMutableArray because don't the filtered places change based on what you type? Does this not error?